packages/angular/angular.json

```diff
@@ -86,8 +86,8 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "7mb",
-                  "maximumError": "8mb"
+                  "maximumWarning": "8mb",
+                  "maximumError": "9mb"
```

Comment:
why this needs to be raised?

Reply:
I think it's cos we have started using scheduler module.